packages/firestore/ios/RNFBFirestore/RNFBFirestoreQuery.m

```diff
@@ -58,6 +58,8 @@ - (void)applyFilters {
 
     if ([operator isEqualToString:@"EQUAL"]) {
       _query = [_query queryWhereFieldPath:fieldPath isEqualTo:value];
+    } else if([operator isEqualToString:@"NOT_EQUAL"]) {
```

Comment:
Space after if here is the only thing I can see. This is sweet!

Reply:
good catch 👍

Reply:
I am *ace* at whitespace haha. Does not quite make up for objective-c being near worthless but we'll just skip that